.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
OK I am going to keep looking. If I can't figure it out after a few hours, I'll just remove the test (the command seems to work fine on several environments where I tested).